components/properties/src/sets.rs

```diff
@@ -1442,46 +1446,35 @@ where
 // Enumerated property getter fns
 //
 
+fn get_set_for_enum_prop_val<D, T>(provider: &D, resc_key: ResourceKey, prop_val: T) -> UnisetResult
```

Comment:
I think we don't want this function. If people want to get a set for an enum property value, they should load the enum property and then call functions on the CPT.